rsk-stackoverflow-stats/index.js

```diff
@@ -0,0 +1,102 @@
+#! /usr/bin/env node
+import axios from 'axios';
+import colors from 'colors';
+
+colors.enable();
+
+// fetches all items returned by the specified API URL.
+// repeats until it responds with `has_more: false`
+async function fetchAllItems(url) {
+  let itemList = [];
+  const fetchSequentially = async (pageNumber = 1) => {
+    // add pagination to the URL
+    const apiUrl = `${url}&page=${pageNumber}`;
+    const response = await axios.get(apiUrl);
+    const { items } = response.data;
+    const hasMore = response.data.has_more;
+    if (
+      [items, hasMore].some((prop) => prop === undefined) ||
+      !Array.isArray(items)
+    ) {
+      throw new Error('The API returns data in an unknown format');
+    }
+    itemList = [...itemList, ...items];
+    if (hasMore) {
+      await fetchSequentially(pageNumber + 1);
+    }
+  };
+  await fetchSequentially();
+  return itemList;
+}
+
+async function getQuestions() {
+  const apiUrl = `https://api.stackexchange.com/2.3/search/advanced?pagesize=100&order=desc&sort=creation&tagged=rsk&site=stackoverflow&filter=!Oev7Wya51xkPXz3rxkrnxh_FQIK9DTSdjn0iuhbW(kx`;
```

Comment:
this line is a bit long... so difficult to maintain in future.
try splitting into multiple lines perhaps using `array.join('')`|
... or better yet, have an object/ array of key-value pairs,
and construct the query params from that, then concat with the base URL.